test-cypress/integration/explore/similarity.spec.js

```diff
@@ -238,6 +246,177 @@ describe('Similarity screen validation', () => {
         });
     });
 
+    context('Confirmations when try to change repository', () => {
```

Comment:
Please, split the tests in these contexts in separate specs

Reply:
Done